property-service/src/main/java/org/egov/ps/validator/PropertyValidator.java

```diff
@@ -63,11 +68,39 @@ private void validateOwner(PropertyRequest request, Map<String, String> errorMap
 		Optional<Property> property_Optional = request.getProperties().stream().filter(p -> !CollectionUtils.isEmpty(p.getPropertyDetails().getOwners()))
 				.findAny();
 		if (property_Optional.isPresent()) {
+			
```

Comment:
This signature of this method should change to accept
```java
private void validateDocumentsOnType(RequestInfo requestInfo, String tenantId, List<Document> documents, Map<String, String> errorMap, String code) {
```
We can then reuse the method for other types of document validtions also. Please look at 
https://github.com/Transerve-PwC/property-services/pull/157/files#diff-eb85ceb59d14e981e6824251ddd6151cR1053

Reply:
@rajiv-transerve  once first two comment clear then will check with this one .

Reply:
@rajiv-transerve  added  method , kindly check and let me know

Reply:
@rajiv-transerve  resolved